### PR TITLE
Support Upload of Zip files

### PR DESCRIPTION
**Pull request by davidbirchwork**
as per Issue #5 

Acceptance test:
> Run code with S3 credentials 
> Create a zip file with files in root and folders with files in them 
> Upload choosing Service type to be Archive 
> Check on S3 that your files exist.

### CODE REVIEW

File: packages/ove-asset-manager/src/OVE.Service.AssetManager/Domain/S3AssetFileOperations.cs [hunk 1]
```diff
@@ -157,8 +209,8 @@ public class S3AssetFileOperations : IAssetFileOperations {
             // filter to valid chars 
             Regex r = new Regex("^[a-zA-Z0-9]+$");
             input = input.Where(l => r.IsMatch(l.ToString())).Aggregate("", (acc, c) => acc + c);
-
-            return "."+input;
+          
+            return "." + input
```
[the remark this answers is not in the document]
Comment (davidbirchwork) [reply packages/ove-asset-manager/src/OVE.Service.AssetManager/Domain/S3AssetFileOperations.cs]: Thanks for the fix, sorry about that, my typo when using gits website based merge conflict tool ….

**Comment by davidbirchwork** [comment]
Appending fix for https://github.com/ove/ove-asset-services/issues/2

**Comment by jamesscottbrown** [comment]
@senakafdo and @oserban, I think that your concerns have mostly been addressed, and the only thing remaining to be done is exposing the URL of the uploaded assets. Is this correct?

**Comment by davidbirchwork** [comment]
thanks @jamesscottbrown exposing the URLs will be coming in an hour or so

**Comment by senakafdo** [comment]
The implementation works in general - tested for an archive from a VIS project at the DSI. However, it fails with a generic archive - I will raise a separate issue for this, if I can consistently reproduce the bug.

**Pull request merged**